environments/stg/stg.tfvars

```diff
@@ -234,14 +234,12 @@ ldap_vms = {
     subnet_name           = "backend"
     install_xdr_agent     = true
     install_xdr_collector = false
-    install_docker        = false
```

Comment:
We need these, this is the replacement for the `resource "azurerm_virtual_machine_extension" "install_docker" {` resource which is being removed.